themes/default/layouts/page/pricing.html

```diff
@@ -1467,7 +1467,7 @@ <h2 class="text-center">Compare editions</h2>
                 <div class="compare-plans-table-row-container">
                     <div class="compare-plans-table-row-item">
                         <span
-                            ><a class="compare-table-link" href="{{ relref . " /docs/intro/pulumi-service/organization-access-tokens/" }}"
+                            ><a class="compare-table-link" href="{{ relref . " /docs/intro/pulumi-service/organization-access-tokens" }}"
```

Comment:
Is the leading space intentional?

`" /docs/..."`

Reply:
im not sure, they are like that throughout this page.

Reply:
The leading spaces, and the newlines inside elements, look weird but don't affect the output -- links still work so far as I can see in the preview build.

At some point it would be good to er, regularise the markup in here (and elsewhere?). I think some of it is `yarn prettier` thinking that angle brackets dangling in space are better than long lines. The leading spaces might just be harmless copypasta.